production-ready.md

```diff
@@ -0,0 +1,52 @@
+# Production ready
+
+This document defines our standard for production ready applications and
+changes from an *operations* standpoint. The point at which the *product* is
+ready can vary highly from project to project.
+
+## Overview
+
+### Applications
+
+At the application scope, we adhere closely to Heroku's
+[Twelve-Factor App](http://12factor.net/) guidelines as a baseline for
+production readiness. The "Production checklist" lists out specific
+implementation guidelines for making applications production ready.
+
+### New code changes
+
+Any time a new changeset is found to impede production readiness (for example,
+a changeset might introduce functionality that, by design, obscures logging or
+fails silently in a production setting), it's not a production ready change,
+and should not be promoted in the release pipeline. Additionally, for any code
+change to become "production ready", it must be verified working in the context
+of a controlled pre-production environment identical to production.
+
+## "Production readiness" checklists
+
+### Required features:
+
+- [ ] Does it support configuration via environment variables?
+- [ ] Does it support configuration via flatfile (e.g. YAML)?
+- [ ] Does it have a sane default configuration (e.g. as close to a realistic production configuration as possible)?
+- [ ] Does it run on parallel HTTP workers? (e.g. uWSGI, Gunicorn)?
+- [ ] Does it log to standard out/error?
+- [ ] Does it log to syslog?
```

Comment:
Same question here. Is this an either/or choice, or a both? (It looks like both)